base/associative.jl

```diff
@@ -87,6 +89,8 @@ eachindex(a::Associative) = KeyIterator(a)
     values(a::Associative)
 
 Return an iterator over all values in a collection.
+`keys(Dict)` and `values(Dict)` both iterate the `Dict`
+ and return the elements in the same order.
 `collect(values(d))` returns an array of values.
```

Comment:
Would make sense to copy the same "Since the keys are..." sentence as for `keys` too.